.controlplane/Dockerfile_from_base

```diff
@@ -0,0 +1,21 @@
+# This Dockerfile should be used for a faster image build on top of the base
+# image from Docker hub (shakacode/rwrt-base:latest). To this end, uncomment
+# the line for the custom dockerfile in the controlplane.yml file.
+
+FROM shakacode/rwrt-base:latest
```

Comment:
This is going to confuse people.

Reply:
I understand.
Perhaps we can move these files into `.shakacode` folder which will be the place for keeping files only related to our development tasks.